common/device_parser.cc

```diff
@@ -25,7 +25,7 @@
 #include <string>
 #include <iostream>
 
-// Example IMAP Client ID string (Apple Mail on MacOS):
+// Example IMAP Client ID string (Apple Mail on macOS):
```

Comment:
the story of Apple's operating system brand names is amusing, but quite long. Happy to provide references for the curious (it's basically a series of links to wikipedia articles chronicling the tale).